Sketchfab-Exporter.pyp

```diff
@@ -51,64 +46,80 @@ except ImportError, err:
         zf.extractall(pythonPath)
         zf.close()
     except Exception, err:
-        gui.MessageDialog("Unable to install necessary files. Please check the help for information on manual module installation.", c4d.GEMB_OK)
+        gui.MessageDialog("Unable to install necessary files. \
```

Comment:
I don't find this very nice

```python
gui.MessageDialog("Unable to install necessary files. "
                                "Please check the help for information on manual module installation.", c4d.GEMB_OK)
```
should work and I tend to prefer this formatting